app/presenters/provider/admin/user/access_tokens_new_presenter.rb

```diff
@@ -7,7 +7,7 @@ def initialize(provider)
   end
 
   def provider_timezone_offset
-    @timezone.utc_offset
+    @timezone.now.utc_offset
```

Comment:
It is not immediately clear to me what contexts this is used in. Is it possible that one may expect the offset in general, not right at the moment?

Reply:
> The user should select time in their browser's timezone (or provider's time zone at worst 🤔) and then the server should respect that later either by storing the time zone or using UTC, depending on the use case.

That's what the code does.

> Ideally though, the time picked by user should have the actual timezone and then backend respect that.

The user picks a date in their time zone, but that instant is stored in DB in UTC. Is not that OK? I don't understand what you mean.

Reply:
@akostadinov I made some changes, is this better?

<img width="587" height="294" alt="image" src="https://github.com/user-attachments/assets/bd27af36-3205-4327-8327-8e26c15b84d9" />